src/perform_step/low_order.jl

```diff
@@ -229,3 +229,127 @@ end
 
   end
 end
+
+@muladd function perform_step!(integrator,cache::RKMil_GeneralConstantCache,f=integrator.f)
+  @unpack t,dt,uprev,u,W,p = integrator
+  Wik = cache.WikJ
+  dW = W.dW
+
+  WikJ = get_iterated_I!(dW, Wik)
+
+  if alg_interpretation(integrator.alg) == :Ito
+    if typeof(dW) <: Number || is_diagonal_noise(integrator.sol.prob)
+      WikJ = WikJ .- 1//2 .* dt
+    else
+      WikJ -= 1//2 .* dt .* Eye{eltype(dW)}(length(dW))
```

Comment:
Are you sure about the identity matrix here?

Reply:
It was working for the normal RKMil_Commute, so I don't see why it should be a problem here.

Reply:
oh yes, now I remember why it's there.